apps/api/src/domain/models/vocabulary-list/errors/index.ts

```diff
@@ -3,3 +3,6 @@ export * from './cannot-have-two-filter-properties-with-the-same-name.error';
 export * from './duplicate-label-for-vocabulary-list-filter-property-value.error';
 export * from './duplicate-value-for-vocabulary-list-filter-property-value.error';
 export * from './duplicate-vocabulary-list-name.error';
+// TODO make sure the imports are correct
```

Comment:
Do this. All should be barrel exports and imported from base directories not nested directories.

Reply:
I fixed the exports but we should fix the imports as we come across them.